moto/ec2/responses/security_groups.py

```diff
@@ -239,6 +247,39 @@ def update_security_group_rule_descriptions_egress(self):
     </tagSet>
 </CreateSecurityGroupResponse>"""
 
+DESCRIBE_SECURITY_GROUP_RULES_RESPONSE = """
+<DescribeSecurityGroupRulesResponse xmlns="http://ec2.amazonaws.com/doc/2016-11-15/">
+  <requestId>2baaa089-3113-4af5-a1cc-c3af1abe7c6e</requestId>
+  <securityGroupRuleSet>
+    <item>
```

Comment:
You can add multiple rules to the XML using a for-loop. As an example:
```
{% for rule in rules %}
<item>
  <cidrIpv4>{{ rule.ipv4 }}</cidrIpv4>
  ...
</item>
{% endfor %}
```
See the `CREATE_SECURITY_GROUP_RESPONSE`-template as another example, that has a similar for-loop.